doc/admin/troubleshooting.md

```diff
@@ -36,6 +36,7 @@ server operations:
 |Control Plane|control_log_file|/tmp/daos_control.log|
 |Data Plane|log_file|/tmp/daos_server.log|
 |[Privileged Helper](https://daos-stack.github.io/admin/deployment/#elevated-privileges)|helper_log_file|/tmp/daos_admin.log|
+|agent|log_file|/tmp/daos_agent.log|
```

Comment:
This list of config parameters is for the daos_server.yml file, so it's a little confusing to add the agent-side parameter here.

Reply:
Should it be more generic?  The section is on Log files and there is a daos_agent logfile.    Or maybe add a column for the configuration file that defines the log file?

Reply:
Oh, I see this is for release/0.9. Well, what I was pointing out is that immediately before this table, there is the text: "On the server side, there are three log files created as part of normal server operations:" ... So now we've got the agent log mixed in with the server logs.

Reply:
I will remove the daos_agent.log from the table